package.json

```diff
@@ -43,6 +43,7 @@
 		"frontend:start": "yarn dev:frontend",
 		"g": "yarn generate",
 		"generate": "hygen",
+		"lint": "yarn workspace @highlight-run/frontend lint",
```

Comment:
could this just be a `yarn turbo run lint` to make this forward-compatible for other workspaces adding the command?